x-pack/solutions/observability/plugins/observability_ai_assistant_management/public/routes/components/knowledge_base_bulk_import_flyout.tsx

```diff
@@ -54,6 +54,7 @@ export function KnowledgeBaseBulkImportFlyout({ onClose }: { onClose: () => void
 
     try {
       entries = elements.map((el) => JSON.parse(el));
+      mutateAsync({ entries }).then(onClose);
```

Comment:
you'd need to `await` this presumably? and I think you can remove the `.then(onClose)`.